api.raml

```diff
@@ -104,11 +208,55 @@ schemas:
         description: |
           The given credentials were NOT acceptable/valid (i.e. - the
           authentication failed).
+        body:
+          application/json:
+            schema: Error
+            example: |
+              {
+                "name": "Bad Request",
+                "message": "Some error message about an invalid login.",
+                "code": 0,
+                "status": 400
```

Comment:
Good catch. I'm fine with `422` for this. I'll go ahead and change it unless you object.